src/restaurant.js

```diff
@@ -66,11 +52,67 @@
 // const orderFromMenu = (request) => // Lógica que adiciona à chave `consumption` de `restaurant` a string recebida no parâmetro `request`. Essa função deve ser associada à chave `order` de `restaurant`
 // ```
 // Agora faça o TESTE 6 no arquivo `tests/restaurant.spec.js`.
-
+// Done
 //------------------------------------------------------------------------------------------
 
 // PASSO 4: Adicione ao objeto retornado por `createMenu()` uma chave `pay` com uma função que varre todo os itens de `objetoRetornado.consumption`, soma o preço de todos checando-os no menu e retorna o valor somado acrescido de 10%. DICA: para isso, você precisará varrer tanto o objeto da chave `food` quanto o objeto da chave `drink`.
 
-const createMenu = () => {};
-
+function orderFromMenu(request) {
+  this.consumption.push(request);
+}
+
+function payMenu() {
+  const foodDrink = Object.assign(
+    {},
+    this.fetchMenu.food,
+    this.fetchMenu.drinks,
```

Comment:
3. Por último, a gente ia ter que tirar o this e colocar objeto no final, como no passa 2
```suggestion
    objeto.fetchMenu.food,
    objeto.fetchMenu.drinks,
```